schemas.yaml

```diff
@@ -574,6 +563,76 @@ schemas:
           ipAddress: 192.168.1.13
           subnetMask: 255.255.255.0
           standardGateway: 192.168.1.1
+
+  voltageObject:
+    type: object
+    properties:
+      value:
+        type: number
+        minimum: 0
+        example: 24
+      unit:
+        type: string
+        minLength: 1
+        maxLength: 32
+        example: "V"
+  currentObject:
+    type: object
+    properties:
+      value:
+        type: number
+        minimum: 0
+        example: 0.5
+      unit:
+        type: string
+        minLength: 1
+        maxLength: 32
+        example: "A"
+  gatewayMonitor:
+    type: object
+    properties:
+      voltage:
+        type: object
+        properties:
+          primaryVoltage:
+            $ref: "#/schemas/voltageObject"
+          secondaryVoltage:
+            $ref: "#/schemas/voltageObject"
+      current:
+        type: object
+        properties:
+          primaryCurrent:
+            $ref: "#/schemas/currentObject"
+          secondaryCurrent:
+            $ref: "#/schemas/currentObject"
```

Comment:
The "voltage", "voltage.primaryVoltage",  "voltage.secondaryVoltage", "current", "current.primaryCurrent" and "current.secondaryCurrent" can be mandatory.